components/bio-formats-plugins/src/loci/plugins/util/ROIHandler.java

```diff
@@ -311,19 +337,48 @@ else if(ijRoi.getTypeAsString().matches("Rectangle")){//Check if its a Rectangle
 
     // -- Helper methods --
 
+    private static void storeText(TextRoi roi, MetadataStore store, int roiNum, int shape) {
+
+        store.setLabelX(roi.getPolygon().getBounds().getX(), roiNum, shape);
+        store.setLabelY(roi.getPolygon().getBounds().getY(), roiNum, shape);
+
+        store.setLabelText(roi.getText().trim(), roiNum, shape);
+        if (roi.getStrokeWidth() > 0) {
+            store.setLabelStrokeWidth( new Length((roi.getStrokeWidth()), UNITS.PIXEL), roiNum, shape);
+        }
+        if (roi.getStrokeColor() != null) {
+            store.setLabelStrokeColor(new ome.xml.model.primitives.Color(roi.getStrokeColor().getColorSpace().getType()) , roiNum, shape);
```

Comment:
Should I append all the getRed,getGreen,getBlue and getAlpha values to form one integer value, that can be parsed to ome.xml.model.primitives.Color(Integer)?
```
    private static ome.xml.model.primitives.Color toOMExmlColor(java.awt.Color color){
        
        ome.xml.model.primitives.Color test = new ome.xml.model.primitives.Color(test1);
        return test ;
        
    }
```